diff --git a/tika-server/src/test/java/org/apache/tika/server/TikaResourceTest.java b/tika-server/src/test/java/org/apache/tika/server/TikaResourceTest.java
index 60ff70910..25d88fbea 100644
--- a/tika-server/src/test/java/org/apache/tika/server/TikaResourceTest.java
+++ b/tika-server/src/test/java/org/apache/tika/server/TikaResourceTest.java
@@ -110,24 +110,45 @@ public class TikaResourceTest extends CXFTestBase {
 		assertEquals(UNPROCESSEABLE, response.getStatus());
 	}
 
-    @Test
-    public void testSimpleWordHTML() throws Exception {
-        Response response = WebClient.create(endPoint + TIKA_PATH)
-                .type("application/msword")
-                .accept("text/html")
-                .put(ClassLoader.getSystemResourceAsStream(TEST_DOC));
-        String responseMsg = getStringFromInputStream((InputStream) response
-                .getEntity());
-        assertTrue(responseMsg.contains("test"));
-    }
-
-    @Test
-    public void testPasswordXLSHTML() throws Exception {
-        Response response = WebClient.create(endPoint + TIKA_PATH)
-                .type("application/vnd.ms-excel")
-                .accept("text/html")
-                .put(ClassLoader.getSystemResourceAsStream("password.xls"));
-
-        assertEquals(UNPROCESSEABLE, response.getStatus());
-    }
+  @Test
+  public void testSimpleWordHTML() throws Exception {
+      Response response = WebClient.create(endPoint + TIKA_PATH)
+              .type("application/msword")
+              .accept("text/html")
+              .put(ClassLoader.getSystemResourceAsStream(TEST_DOC));
+      String responseMsg = getStringFromInputStream((InputStream) response
+              .getEntity());
+      assertTrue(responseMsg.contains("test"));
+  }
+
+  @Test
+  public void testPasswordXLSHTML() throws Exception {
+      Response response = WebClient.create(endPoint + TIKA_PATH)
+              .type("application/vnd.ms-excel")
+              .accept("text/html")
+              .put(ClassLoader.getSystemResourceAsStream("password.xls"));
+
+      assertEquals(UNPROCESSEABLE, response.getStatus());
+  }
+
+  @Test
+  public void testSimpleWordXML() throws Exception {
+    Response response = WebClient.create(endPoint + TIKA_PATH)
+      .type("application/msword")
+      .accept("text/xml")
+      .put(ClassLoader.getSystemResourceAsStream(TEST_DOC));
+    String responseMsg = getStringFromInputStream((InputStream) response
+      .getEntity());
+    assertTrue(responseMsg.contains("test"));
+  }
+
+  @Test
+  public void testPasswordXLSXML() throws Exception {
+    Response response = WebClient.create(endPoint + TIKA_PATH)
+      .type("application/vnd.ms-excel")
+      .accept("text/xml")
+      .put(ClassLoader.getSystemResourceAsStream("password.xls"));
+
+    assertEquals(UNPROCESSEABLE, response.getStatus());
+  }
 }
