diff --git a/tika-parsers/src/test/java/org/apache/tika/parser/image/PSDParserTest.java b/tika-parsers/src/test/java/org/apache/tika/parser/image/PSDParserTest.java
index 3fe4a6d9d..a6dcb078b 100644
--- a/tika-parsers/src/test/java/org/apache/tika/parser/image/PSDParserTest.java
+++ b/tika-parsers/src/test/java/org/apache/tika/parser/image/PSDParserTest.java
@@ -25,6 +25,8 @@ import org.xml.sax.helpers.DefaultHandler;
 
 import junit.framework.TestCase;
 
+import static junit.framework.Assert.assertEquals;
+
 public class PSDParserTest extends TestCase {
 
     private final Parser parser = new PSDParser();
@@ -43,4 +45,19 @@ public class PSDParserTest extends TestCase {
         assertEquals("51", metadata.get(Metadata.IMAGE_LENGTH));
         assertEquals("8", metadata.get(Metadata.BITS_PER_SAMPLE));
     }
+    
+    /**
+     * Tests a very basic file, without much metadata,
+     *  where some of the data lengths are padded to be even
+     */
+    public void testOddPSD() throws Exception {
+        Metadata metadata = new Metadata();
+        metadata.set(Metadata.CONTENT_TYPE, "image/x-psd");
+        InputStream stream =
+            getClass().getResourceAsStream("/test-documents/testPSD2.psd");
+        parser.parse(stream, new DefaultHandler(), metadata, new ParseContext());
+        assertEquals("69", metadata.get(Metadata.IMAGE_WIDTH));
+        assertEquals("70", metadata.get(Metadata.IMAGE_LENGTH));
+        assertEquals("8", metadata.get(Metadata.BITS_PER_SAMPLE));
+    }
 }
