diff --git a/tika-parsers/src/test/java/org/apache/tika/parser/microsoft/ooxml/OOXMLParserTest.java b/tika-parsers/src/test/java/org/apache/tika/parser/microsoft/ooxml/OOXMLParserTest.java
index 1a716ade5..f0c681040 100644
--- a/tika-parsers/src/test/java/org/apache/tika/parser/microsoft/ooxml/OOXMLParserTest.java
+++ b/tika-parsers/src/test/java/org/apache/tika/parser/microsoft/ooxml/OOXMLParserTest.java
@@ -897,4 +897,10 @@ public class OOXMLParserTest extends TikaTest {
         assertTrue(i < j);
         assertTrue(j < k);
     }
+  
+    // TIKA-1006
+    public void testWordNullStyle() throws Exception {
+      String xml = getXML("testWORD_null_style.docx").xml;        
+      assertContains("Test av styrt dokument", xml);
+    }
 }
