diff --git a/tika-app/src/test/java/org/apache/tika/cli/TikaCLITest.java b/tika-app/src/test/java/org/apache/tika/cli/TikaCLITest.java
index d5366178f..d6b5ee9d8 100644
--- a/tika-app/src/test/java/org/apache/tika/cli/TikaCLITest.java
+++ b/tika-app/src/test/java/org/apache/tika/cli/TikaCLITest.java
@@ -215,4 +215,14 @@ public class TikaCLITest extends TestCase{
         assertTrue(content.contains("sheetNames: Logical Sheet 3"));
         assertTrue(content.contains("sheetNames: Sheet 4"));
     }
+
+    // TIKA-1031
+    public void testZipWithSubdirs() throws Exception {
+        String[] params = {"-z", resourcePrefix + "testWithSubdirs.zip"};
+        new File("subdir/foo.txt").delete();
+        new File("subdir").delete();
+        TikaCLI.main(params);
+        String content = outContent.toString();
+        assertTrue(content.contains("Extracting 'subdir/foo.txt'"));
+    }
 }
