diff --git a/tika-parsers/src/test/java/org/apache/tika/parser/microsoft/WordParserTest.java b/tika-parsers/src/test/java/org/apache/tika/parser/microsoft/WordParserTest.java
index c3770011b..a1533d90f 100644
--- a/tika-parsers/src/test/java/org/apache/tika/parser/microsoft/WordParserTest.java
+++ b/tika-parsers/src/test/java/org/apache/tika/parser/microsoft/WordParserTest.java
@@ -332,6 +332,6 @@ public class WordParserTest extends TikaTest {
     }
 
     public void testTabularSymbol() throws Exception {
-        assertContains("one\ntwo", getXML("testWORD_tabular_symbol.doc").xml);
+        assertContains("one two", getXML("testWORD_tabular_symbol.doc").xml.replaceAll("\\s+", " "));
     }
 }
