diff --git a/tika-parsers/src/test/java/org/apache/tika/parser/xml/DcXMLParserTest.java b/tika-parsers/src/test/java/org/apache/tika/parser/xml/DcXMLParserTest.java
index 087966e12..88f8de8ec 100644
--- a/tika-parsers/src/test/java/org/apache/tika/parser/xml/DcXMLParserTest.java
+++ b/tika-parsers/src/test/java/org/apache/tika/parser/xml/DcXMLParserTest.java
@@ -18,15 +18,14 @@ package org.apache.tika.parser.xml;
 
 import java.io.InputStream;
 
-import junit.framework.TestCase;
-
+import org.apache.tika.TikaTest;
 import org.apache.tika.metadata.Metadata;
 import org.apache.tika.metadata.TikaCoreProperties;
 import org.apache.tika.sax.BodyContentHandler;
 import org.xml.sax.ContentHandler;
 import org.xml.sax.helpers.DefaultHandler;
 
-public class DcXMLParserTest extends TestCase {
+public class DcXMLParserTest extends TikaTest {
 
     public void testXMLParserAsciiChars() throws Exception {
         InputStream input = DcXMLParserTest.class.getResourceAsStream(
@@ -92,4 +91,9 @@ public class DcXMLParserTest extends TestCase {
         }
     }
 
+    // TIKA-1048
+    public void testNoSpaces() throws Exception {
+      String text = getXML("testXML2.xml").xml;
+      assertFalse(text.contains("testSubject"));
+    }
 }
