diff --git a/tika-app/src/test/java/org/apache/tika/cli/TikaCLITest.java b/tika-app/src/test/java/org/apache/tika/cli/TikaCLITest.java
index d6b5ee9d8..fa305a404 100644
--- a/tika-app/src/test/java/org/apache/tika/cli/TikaCLITest.java
+++ b/tika-app/src/test/java/org/apache/tika/cli/TikaCLITest.java
@@ -224,5 +224,8 @@ public class TikaCLITest extends TestCase{
         TikaCLI.main(params);
         String content = outContent.toString();
         assertTrue(content.contains("Extracting 'subdir/foo.txt'"));
+        // clean up. TODO: These should be in target.
+        new File("subdir/foo.txt").delete();
+        new File("subdir").delete();
     }
 }
