diff --git a/tika-parsers/src/test/java/org/apache/tika/mime/TestMimeTypes.java b/tika-parsers/src/test/java/org/apache/tika/mime/TestMimeTypes.java
index 2260d5b32..5b2b8b494 100644
--- a/tika-parsers/src/test/java/org/apache/tika/mime/TestMimeTypes.java
+++ b/tika-parsers/src/test/java/org/apache/tika/mime/TestMimeTypes.java
@@ -632,6 +632,10 @@ public class TestMimeTypes extends TestCase {
     public void testEmlx() throws IOException {
         assertTypeDetection("testEMLX.emlx", "message/x-emlx");
     }
+    
+    public void testGroupWiseEml() throws Exception {
+        assertTypeDetection("testGroupWiseEml.eml", "message/rfc822");
+    }
 
     /** Test getMimeType(byte[]) */
     public void testGetMimeType_byteArray() throws IOException {
